core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -110,6 +110,20 @@ public static boolean isCarbonDataFile(String fileNameWithPath) {
     return false;
   }
 
+  /**
+   * check if it is carbon partitionmap file matching extension
```

Comment:
change to 
> Return true if the `fileNameWithPath` ends with partition map file extension name

remove @param and @return

Reply:
ok